app/server/utils/user.py

```diff
@@ -224,6 +224,9 @@ def create_transfer_account_user(first_name=None, last_name=None, preferred_lang
         if current_app.config['AUTO_APPROVE_TRANSFER_ACCOUNTS'] and not is_self_sign_up:
             transfer_account.approve_and_disburse()
 
+        db.session.add(transfer_account)
```

Comment:
had to add this or else `transfer_account.id` is `None` and so `default_transfer_account_id` isn't set correctly... but we're not supposed to commit early...? @enjeyw

Reply:
this _should_ work:
```

db.session.add(transfer_account)
user.default_transfer_account = transfer_account

```

Reply:
this should work:
```

db.session.add(transfer_account)
user.default_transfer_account = transfer_account

```

Reply:
doesn't seem to work, still getting None on line `transfer_account = TransferAccount.query.get(user.default_transfer_account_id)` in `test_directory_listing`